packages/embed-grid/src/App.tsx

```diff
@@ -61,6 +71,8 @@ function App(): JSX.Element {
   );
   const canCopy = searchParams.get('canCopy') != null;
   const canDownloadCsv = searchParams.get('canDownloadCsv') != null;
+  const type = (searchParams.get('type') ??
+    dh.VariableType.TABLE) as VariableTypeUnion;
 
   useEffect(
     function initializeApp() {
```

Comment:
The majority of this hook could probably be pulled into a custom hook like `const connection = useConnection({ url, credentials, onSuccess, onError })` or something. But that's for another day